kuma/wiki/jinja2/wiki/includes/spam_error.html

```diff
@@ -14,3 +14,10 @@
     contact the site administrators</a>.
     {% endtrans %}
 </p>
+{% if perms.wiki.add_revisionakismetsubmission %}
```

Comment:
I'm wondering if I don't have Akismet running locally.... I can't get the spam to get caught although I've created this new test account locally. I'll try to figure out a way around it and keep working!